cvat-ui/src/index.tsx

```diff
@@ -123,10 +129,13 @@ function mapDispatchToProps(dispatch: any): DispatchToProps {
         resetErrors: (): void => dispatch(resetErrors()),
         resetMessages: (): void => dispatch(resetMessages()),
         loadOrganization: (): void => dispatch(activateOrganizationAsync()),
-        initInvitations: (): void => dispatch(getInvitationsAsync({ page: 1 }, true)),
-        initRequests: (): void => dispatch(getRequestsAsync({ page: 1 })),
+        initInvitations: (): void => dispatch(getInvitationsAsync({ page: 1, pageSize: 10 }, true)),
+        initRequests: (): void => dispatch(getRequestsAsync({ page: 1, pageSize: 10 })),
```

Comment:
Should be initialized from InvitationsState/RequestsState instead of hard-coding 10 here.

Reply:
BTW, `pageSize` is missing in default value for InvitationdState and typescript complaints about it